app/src/main/java/com/example/scoutconcordia/DES.java

```diff
@@ -172,15 +174,18 @@ public void unlockFile(String filename)
         
         try
         {
+            // THIS IS NOT WORKING
```

Comment:
Definitely shouldn't merge a PR with this comment

Reply:
outdated.